### PR TITLE
Networking

### PR DESCRIPTION
**Pull request by strdr4605**

### CODE REVIEW

File: Assets/PlayerObject.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    [Command]
+    void CmdMoveUnit(Vector3 move) {
+        if(myPlayerUnit == null) {
+            return;
+        }
+
+
+        myPlayerUnit.transform.position += move * speed * Time.deltaTime;
```
Comment (curlyfingers) [inline Assets/PlayerObject.cs:51]: Please rotate sprite in direction of the movement

File: Assets/PlayerObject.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		
+	}
+
+	// Update is called once per frame
+	void Update () {
+        if( isLocalPlayer == false) {
+            return;
+        }
+
+        var move = new Vector3(Input.GetAxis("Horizontal"), Input.GetAxis("Vertical"), 0);
```
Comment (curlyfingers) [inline Assets/PlayerObject.cs:30]: I slightly understand how it works with the joystick, what about keyboard input?

Comment (strdr4605) [reply Assets/PlayerObject.cs:30]: There are some commands for Networking, related to key "S" and when the player hit this key he disconnects from the server.

File: ProjectSettings/EditorBuildSettings.asset [hunk 3]
```diff
@@ -5,7 +5,10 @@ EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
   m_Scenes:
-  - enabled: 1
-    path: Assets/Scenes/SampleScene.unity
+  - enabled: 0
+    path: Assets/Scenes/DesignerScene.unity
     guid: 2cda990e2423bbf4892e6590ba056729
+  - enabled: 1
+    path: Assets/Scenes/DevScene.unity
```
Comment (curlyfingers) [inline ProjectSettings/EditorBuildSettings.asset:12]: correct me if i am wrong, but it builds from dev scene?
I think we should build from designer scene

Comment (strdr4605) [reply ProjectSettings/EditorBuildSettings.asset:12]: Yes, but for testing I needed to build from DevScene

**Comment by strdr4605** [comment]
I have some trouble resolving this conflicts, I did not modify those Assets, do you know how to resolve them.

**Comment by curlyfingers** [comment]
try backing up relevant ones and then removing them.
merging unity projects seems to be a huge pain

**Pull request closed without merging**